locales/de/LC_MESSAGES/volto.po

```diff
@@ -1481,6 +1526,11 @@ msgstr "Link"
 msgid "Link more"
 msgstr "'Mehr' Link"
 
+#: config/Views
+# defaultMessage: Link redirect view
+msgid "Link redirect view"
+msgstr "Link-Umleitungsansicht"
```

Comment:
@sneridagh major props for coming up with this beautiful German compound word! :)